components/http_foundation/session_php_bridge.rst

```diff
@@ -16,7 +16,7 @@ However when there really are circumstances where this is not possible, you
 can use a special storage bridge
 :class:`Symfony\\Component\\HttpFoundation\\Session\\Storage\\PhpBridgeSessionStorage`
 which is designed to allow Symfony to work with a session started outside of
-the Symfony Session framework. You are warned that things can interrupt this
+the Symfony session framework. You are warned that things can interrupt this
```

Comment:
what do we mean by this? There is no session framework, nor a component. Do we mean "outside of the HttpFoundatino component"?